### ISSUE TITLE
Term doesn't capture some keys

### ISSUE THREAD

**Issue by babolivier** [alpha14/cozy-term#7]
Hi,
When I can't connect to my server directly through SSH, I use SSH via Term in my Cozy (very useful). However, a few programs I'm using require the use of the "alt" key. For instance, with weechat, I switch between IRC channels via alt+1, alt+2, etc; when I try this in Term's prompt I only get symbols like ☻ or ♥, which prevents me from using this kind of softwares.

**Comment by babolivier** [comment]
Adding to this the backslash ("\") and pipe ("|") keys. Maybe more of them, but I haven't tried every one yet.

**Comment by Shuunen** [comment]
copy/paste \o/

**Comment by babolivier** [comment]
Well, I think that's another issue as the key required for copy/paste are captured by the application (you can do Ctrl+\* commands, and type "C" or "V"). My problem is that, whenever I type "\" of "|" in my terminal, no character appears.

**Comment by Shuunen** [comment]
Yes you can Ctrl+c for example to stop/cancel processes, but Ctrl+v is not handled as paste.

On my side when I type "\" it goes backward and delete my last character.

When I type "|" nothing happens like "@" , alternatively you can use the ascii code but it's not very intuitive.

"|" is Alt+124

**Comment by Zykino** [comment]
An other problem related to "bad keyboard management" is that I have my android keyboard reset to CAPS after each key pressed (using Hacker's Keyboard at least)

**Comment by LM1LC3N7** [comment]
I copy & paste some characters, like `#` or `|`, but I is not really convenient.

Is some found a solution? 

It's too bad that this bug persist, because this module is really, really, really usefull! (to bypass Firewall or network limitations).
